### ISSUE TITLE
Include icon in Windows context menu

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vim/vim#249]
```
What version of the product are you using? On what operating system?
- GVim 7.4 (Aug 10, 2013)
- Windows 7, 64-bit

Please include the vim icon in the Windows context menu when right-clicking on 
a file to edit.  This would make it much easier to find the "Edit with Vim" 
entry, especially on context menus that have numerous other additional menus, 
many of which also already have icons.
```

Original issue reported on code.google.com by `brian.ma...@betteradmin.com` on 18 Aug 2014 at 9:00

**Comment by orev** [comment]
I just updated to 8.0 and this is not included with it.  On Windows 7 64-bit.

**Comment by k-takata** [comment]
It seems that gvimext.dll bundled with the official installer (gvim80.exe) is old. The build timestamp is 2008-08-10.
The nightly builds have newer gvimext.dll: https://github.com/vim/vim-win32-installer/releases

**Comment by orev** [comment]
As of the gvim80-586 installer, this is still not included.  However it is included in the nightly build as of 8.0.0604.

It is also not included in the "Vim without Cream" build as of 8-0-568

**Comment by brammool** [comment]
orev wrote:

> As of the gvim80-586 installer, this is still not included.  However it is included in the nightly build as of 8.0.0604.

Is this a matter if including a more recent gvimext.dll?
Does the new dll still work on MS-Windows XP (the oldest we support)?

> It is also not included in the "Vim without Cream" build as of 8-0-568

-- 
ARTHUR: What are you going to do. bleed on me?
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by orev** [comment]
I tested the 604 build on an XP machine, and it seems to work fine.  However, the icon is a little off, but maybe that's just how XP works.  See attached:

![gvimext-604-winxp](https://cloud.githubusercontent.com/assets/2551899/26608667/d5c893ee-456a-11e7-87e2-6b8b8e4b3bb7.png)
